README.md

```diff
@@ -12,18 +12,34 @@ ROS tutorial for Webots:
   - https://www.cyberbotics.com/doc/guide/using-ros
 
 
-## Acknowledgement
+## Acknowledgements
 
 <a href="http://rosin-project.eu">
-  <img src="http://rosin-project.eu/wp-content/uploads/rosin_ack_logo_wide.png" 
+  <img src="http://rosin-project.eu/wp-content/uploads/rosin_ack_logo_wide.png"
        alt="rosin_logo" height="60" >
 </a></br>
 
 Supported by ROSIN - ROS-Industrial Quality-Assured Robot Software Components.  
 More information: <a href="http://rosin-project.eu">rosin-project.eu</a>
 
-<img src="http://rosin-project.eu/wp-content/uploads/rosin_eu_flag.jpg" 
+<img src="http://rosin-project.eu/wp-content/uploads/rosin_eu_flag.jpg"
      alt="eu_flag" height="45" align="left" >  
 
 This project has received funding from the European Union’s Horizon 2020  
-research and innovation programme under grant agreement no. 732287. 
+research and innovation programme under grant agreement no. 732287.
+
+<br>
+
+<a href="https://opendr.eu/">
+  <img src="https://opendr.eu/wp-content/uploads/2020/01/logo-300x125.png"
+       alt="opendr_logo" height="60" >
+</a></br>
+
+Supported by OpenDR - Open Deep Learning Toolkit for Robotics.  
+More information: <a href="https://opendr.eu/">opendr.eu</a>
+
+<img src="https://opendr.csd.auth.gr/wp-content/uploads/2019/12/Flag_of_Europe-300x200.png"
+     alt="eu_flag" height="45" align="left" >  
+
+This project has received funding from the European Union’s Horizon 2020  
+research and innovation programme under grant agreement no. 871449.
```

Comment:
This file should not be changed

Reply:
As far as I can tell these changes were not introduced by me. I believe they came from this commit [4a52aa2](https://github.com/cyberbotics/webots_ros/pull/63/commits/4a52aa2b89955ae73dcc762e59c8b5e5fb0eca03). Should I still remove these changes?

Reply:
You probably forked from an old version, merging the develop branch into this one should resolve the issue

Reply:
Yes the problem was that I forked the master branch rather than the develop. This has been resolved.